app/handlers/newflow/educator_complete_profile.rb

```diff
@@ -67,6 +60,10 @@ def push_lead_to_salesforce
       end
     end
 
+    def kind_of_educator
+      signup_params.educator_specific_role == 'other' ? signup_params.other_role_name : signup_params.educator_specific_role
```

Comment:
consider putting 'other' as a constant.

consider a method name of `educator_type`? (subtle..just wondering if this sounds better than "kind_of". either way is fine :)

Reply:
does the role need to be stripped? (any chance this role could come in w/ leading or trailing spaces?  it's good to nip this in the bud if so and trim the inputs (always)

Reply:
These are good ideas I would have implemented before hitting the merge button. I can definitely implement them in another PR which I'm currently working on unless you think I should reverse this PR